.github/workflows/build.yml

```
@@ -152,6 +152,6 @@ jobs:
         run: |
           export SINGLE_POD_NAME=$(kubectl get pod -l app=django-mindjunkies-web-deployment -o jsonpath="{.items[0].metadata.name}")
           echo "Running migrations on pod: $SINGLE_POD_NAME"
-          kubectl exec -it $SINGLE_POD_NAME -- bash /app/migrate.sh
+          kubectl exec -i $SINGLE_POD_NAME -- bash /app/migrate.sh
           echo "Collecting static files on pod: $SINGLE_POD_NAME"
-          kubectl exec -it $SINGLE_POD_NAME -- bash /app/collectstatic.sh
\ No newline at end of file
+          kubectl exec -i $SINGLE_POD_NAME -- bash /app/collectstatic.sh
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)